test/cpp/microbenchmarks/bm_call_create.cc

```diff
@@ -609,6 +617,125 @@ typedef Fixture<&grpc_server_load_reporting_filter, CHECKS_NOT_LAST>
+
+  grpc_channel_args channel_args = {args.size(), &args[0]};
+
+  std::vector<const grpc_channel_filter*> filters;
+  if (fixture.filter != nullptr) {
+    filters.push_back(fixture.filter);
+  }
+
+  if (fixture.flags & CHECKS_NOT_LAST) {
+    // Some filters assert they are not the last in the stack, so we add a 
+    // dummy filter after them so they won't complain
+    filters.push_back(&dummy_filter::dummy_filter);
+    label << " #has_dummy_filter";
+  } 
+
+  grpc_core::ExecCtx exec_ctx;
+  size_t channel_size = grpc_channel_stack_size(
+      filters.size() == 0 ? nullptr : &filters[0], filters.size());
+  grpc_channel_stack* channel_stack =
+      static_cast<grpc_channel_stack*>(gpr_zalloc(channel_size));
+  GPR_ASSERT(GRPC_LOG_IF_ERROR(
+      "channel_stack_init",
+      grpc_channel_stack_init(1, FilterDestroy, channel_stack, &filters[0],
+                              filters.size(), &channel_args,
+                              fixture.flags & REQUIRES_TRANSPORT
+                                  ? &dummy_transport::dummy_transport
+                                  : nullptr,
+                              "CHANNEL", channel_stack)));
+  grpc_core::ExecCtx::Get()->Flush();
+  grpc_call_stack* call_stack =
+      static_cast<grpc_call_stack*>(gpr_zalloc(channel_stack->call_stack_size));
+  grpc_millis deadline = GRPC_MILLIS_INF_FUTURE;
+  gpr_timespec start_time = gpr_now(GPR_CLOCK_MONOTONIC);
+  grpc_slice method = grpc_slice_from_static_string("/foo/bar");
+  grpc_call_final_info final_info;
+  grpc_call_element_args call_args;
+  call_args.call_stack = call_stack;
+  call_args.server_transport_data = nullptr;
+  call_args.context = nullptr;
+  call_args.path = method;
+  call_args.start_time = start_time;
+  call_args.deadline = deadline;
+  const int kArenaSize = 4096;
+  call_args.arena = gpr_arena_create(kArenaSize);
+  GRPC_ERROR_UNREF(
+        grpc_call_stack_init(channel_stack, 1, DoNothing, nullptr, &call_args));
+  
+  while (state.KeepRunning()) {
+    GPR_TIMER_SCOPE("BenchmarkCycle", 0);
+    /* Create new payload */
+    grpc_transport_stream_op_batch_payload payload = {};
+    grpc_metadata_batch metadata_batch = {};
+    grpc_metadata_batch_init(&metadata_batch);
+    payload.send_initial_metadata.send_initial_metadata = &metadata_batch;
+    payload.send_initial_metadata.send_initial_metadata_flags = 0;
+    // TODO(hcaseyal): set payload send_initial_metadata peer_string?
+    payload.send_trailing_metadata.send_trailing_metadata = &metadata_batch;
+    payload.recv_initial_metadata.recv_initial_metadata = &metadata_batch;
+    payload.recv_initial_metadata.recv_initial_metadata_ready = nullptr;
+    // TODO(hcaseyal): set recv_initial_metadata peer_string?
+    payload.recv_message.recv_message = nullptr;
+    payload.recv_message.recv_message_ready = nullptr;
+    payload.recv_trailing_metadata.recv_trailing_metadata = nullptr;
+    grpc_transport_stream_stats stats = {};
```

Comment:
Instead of `= {}`, please `memset()` this to all zero.

Reply:
Done